lib/puppet/file_system/windows.rb

```diff
@@ -3,6 +3,14 @@
 
 class Puppet::FileSystem::Windows < Puppet::FileSystem::Posix
 
+  def path_string(path)
+    # attempt to expand
+    # Dir.glob(path.to_s).first || path.to_s
```

Comment:
We have 2 options here... use `Dir.glob` on Windows, which does a bunch of work... or just expand the file the simpler way with the Windows API.